### PR TITLE
Ensure ASLR is off on Linux

### LINKED ISSUE #70: Turn off ASLR if possible
We should turn off address space layout randomisation on platforms that support it.

### PR DESCRIPTION
**Pull request by vext01**
* Check is ASLR is off, if not turn off.
 * More robust checking of CPU governors.

Fixes #70 

OK?

### CODE REVIEW

**Comment by snim2** [comment]
This looks OK to me. What does it do in the presence of `--develop`?

File: krun/platform.py [hunk 1]
```diff
@@ -400,6 +402,7 @@ def check_preliminaries(self):
         self._check_cpu_scaler()
         self._check_perf_samplerate()
         self._check_tickless_kernel()
+        self._check_aslr_disabled()
```
Comment (vext01) [inline krun/platform.py]: The function `check_preliminaries` is called from `krun.py` and is guarded like this:

```
    if not args.develop:                                                        
        platform.check_preliminaries()                                          
    else:                                                                       
        # Needed to skip the use of certain tools and techniques.               
        # E.g. taskset on Linux, and switching user.                            
        warn("Not checking platform prerequisites due to developer mode")       
        platform.developer_mode = True
```

Comment (vext01) [inline krun/platform.py]: So it is skipped if `--develop` is passed.

**Comment by snim2** [comment]
OK, lgtm.

**Pull request merged**